build

```diff
@@ -25,12 +25,20 @@ if [ "$TARGET" == "vm" ]; then
     --arg buildLive false \
     --arg buildDisk false)
 
-  echo
-  echo "Run ./result/bin/run-in-vm to start a VM."
-  echo
-  echo "- unfocus: ctrl-alt-g"
-  echo "- quit: ctrl-alt-q"
-  echo "- tty: ctrl-alt-2, sendkey ctrl-alt-f1, ctrl-alt-1"
+echo -e "
+Run ./result/bin/run-in-vm to start a VM.
+
+QEMU Shortcuts:
+- Grab/release cursor: ctrl-alt-g
+- Quit: ctrl-alt-q
+
+VM shortcuts:
+Switch to QEMU monitor (ctrl-alt-2), activate shortcut via 'sendkey <some-shortcut>', then switch back to QEMU display (ctrl-alt-1).
+- TTY: ctrl-alt-f1
```

Comment:
VT1 is normally disabled, though I was thinking about a debug mode (`playos.debug.enable = true;`) that enables virtual console 1, sets a known password for the root user, maybe adds tmux to system packages; it would have to refuse to compile under certain conditions, so it could never be left on by accident.

Reply:
[Remove TTY shortcut hint as it is normally disabled](https://github.com/dividat/playos/pull/136/commits/3b812b49b8cb286c386ac699f9848418a33c50b6)